service/handlers_test.go

```diff
@@ -181,63 +181,41 @@ func TestCreateProject(t *testing.T) {
 	tests := []test{
 		{
 			name:     "fails to create project when not admin",
-			req:      loadCreateProjectRequest(t, "TestCreateProject/fails_to_create_project_when_not_admin.json"),
+			req:      loadJSON(t, "TestCreateProject/fails_to_create_project_when_not_admin_request.json"),
 			want:     http.StatusUnauthorized,
 			respFile: "TestCreateProject/fails_to_create_project_when_not_admin_response.json",
 			url:      "/projects",
 			method:   "POST",
 		},
 		{
-			name:    "can create project",
-			req:     loadCreateProjectRequest(t, "TestCreateProject/can_create_project.json"),
-			want:    http.StatusOK,
-			asAdmin: true,
-			url:     "/projects",
-			method:  "POST",
-		},
-		{
-			name:    "git repo must be valid",
-			req:     loadCreateProjectRequest(t, "TestCreateProject/git_repo_must_be_valid.json"),
-			want:    http.StatusBadRequest,
-			asAdmin: true,
-			url:     "/projects",
-			method:  "POST",
-		},
-		{
-			name:    "project name must be alphanumeric",
-			req:     loadCreateProjectRequest(t, "TestCreateProject/project_name_must_be_alphanumeric.json"),
-			want:    http.StatusBadRequest,
-			asAdmin: true,
-			url:     "/projects",
-			method:  "POST",
-		},
-		{
-			name:    "project name cannot be less than 4 characters",
-			req:     loadCreateProjectRequest(t, "TestCreateProject/project_name_cannot_be_less_than_4_characters.json"),
-			want:    http.StatusBadRequest,
-			asAdmin: true,
-			url:     "/projects",
-			method:  "POST",
+			name:     "can create project",
+			req:      loadJSON(t, "TestCreateProject/can_create_project_request.json"),
+			want:     http.StatusOK,
+			respFile: "TestCreateProject/can_create_project_response.json",
+			asAdmin:  true,
+			url:      "/projects",
+			method:   "POST",
 		},
 		{
-			name:    "project name cannot be greater than 32 characters",
-			req:     loadCreateProjectRequest(t, "TestCreateProject/project_name_cannot_be_greater_than_32_characters.json"),
-			want:    http.StatusBadRequest,
-			asAdmin: true,
-			url:     "/projects",
-			method:  "POST",
```

Comment:
You'll see lots of tests removed like this. These were moved into unit tests in the requests lib. The handler tests shouldn't have to care about all of these cases. For the most part, the handler tests just needs one bad client validation request to make sure it's returning the proper status, etc

Reply:
Works for me.